src/include/daos_pool.h

```diff
@@ -88,6 +88,7 @@ struct daos_pool_space {
 	uint32_t		ps_padding;
 };
 
+/** Pool rebuild status */
```

Comment:
There's something odd here as Doxygen is seemingly posting to incorrect line numbers, ps_space is off by 6 yet ps_free_min which is only two lines away in the source is accurate.   There is a problem with the GHA/Doxygen job that it's running with the PR merged with master, my other PR I linked before addresses this by having the job checkout the reference but that still doesn't explain this difference.

Perhaps the issue is simply the way Doxygen itself outputs the warnings?

Reply:
In the github page for the GHA Doxygen job the output there seemed to have accurate line numbers. I guess they are off here in this files diff view page? Have fixed up and (force) pushed some changes that I think fixes the issues in these two files.